dir.props

```diff
@@ -251,7 +251,7 @@
     <UAPAOTPackageRuntimePath>$(BinDir)pkg\uapaot\lib</UAPAOTPackageRuntimePath>
 
     <!-- We add extra binplacing for the test shared framework until we can get hardlinking with the runtime directory working on all platforms -->
-    <BinPlaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp'">true</BinPlaceTestSharedFramework>
+    <BinPlaceTestSharedFramework Condition="'$(_bc_TargetGroup)' == 'netcoreapp' or '$(_bc_TargetGroup)' == 'uap'">true</BinPlaceTestSharedFramework>
```

Comment:
We should probably make this condition unique to uap so and have it be conditional on the existence of your test runner payload.